README.md

```diff
@@ -1,15 +1,17 @@
-# gigavibe
-> Discord bot written in skript for audio functionality.
-> Has youtube music, custom (user defined) dj roles. 
+# Gigavibe
+ Discord bot written in skript for audio functionality.
+ Has youtube music, custom (user defined) dj roles. 
 
-# Prerequisites
+## Prerequisites
 > jdk8/jdk11/jdk16 
 > (jdk11 recommended)
 
 > A minecraft server running bukkit/spigot/paper
-> (skigot recommended for long term usage, paper for short term)
+> (spigot recommended for long term usage, paper for short term)
 
-# Installation
-> Move everything in the plugins folder into your minecraft server plugin folder
+## Installation
+If you need help installing any of the servers listed above, refer to the guides for 
+[Spigot](https://www.spigotmc.org/wiki/spigot-installation/), [Bukkit](https://www.wikihow.com/Create-a-Bukkit-Minecraft-Server) and [Paper](https://paper.readthedocs.io/en/latest/server/getting-started.html)
```

Comment:
@ZeNyfh Comment on this please

Reply:
will change this to include a video instead. Will create a batch/shellscript to complete the actions for the user (placing all files into their folders, etc) making the installation alot simpler by making the server not require any proper setting up apart from agreeing to eula and starting the server with a bat/sh file (port forwarding for this is not required for something like this)